app/models/repository.rb

```diff
@@ -25,6 +28,12 @@ def self.license_options
     presence:   { message: "Project title is required."},
     uniqueness: { message: "Project title is already in use.", scope: :user_username}
 
+  validates :share_type,
```

Comment:
I think there's a shortcut for this where you can say `validate :foo, inclusion: ['this', 'that']`